airbyte-integrations/connectors/destination-gcs/src/main/java/io/airbyte/integrations/destination/gcs/util/Consts.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.gcs.util;
+
+public class Consts {
+
+  public static final String EXPECTED_ROLES = "storage.multipartUploads.abort, storage.multipartUploads.create, "
+      + "storage.objects.create, storage.objects.delete, storage.objects.get, storage.objects.list";
+
+  public static final String CHECK_ACTIONS_TMP_FILE_NAME = "test";
+
+  // Uploads this text size will automatically use a multipart upload strategy, while uploads smaller
+  // than this threshold will use
+  // a single connection to upload the whole object even if multipart upload option is ON.
```

Comment:
Is there a documentation about this threshold that will trigger multipart upload? If so, you can paste the link here.

Reply:
Totally fine to ignore this comment if you find this threshold by trail and error.

Reply:
Done, thanks